spk/rutorrent/src/bin/rtorrent-daemon

```diff
@@ -1,22 +1,34 @@
 #!/bin/bash
 
```

Comment:
As `service-setup.sh` now allows passing more than one daemon start|stop|status, could the `rtorrent-daemon` be merged within service-setup?

Reply:
Actually, due to export constraints (playing with `PATH` and `LD_LIBRARY_PATH` it wasn't possible due to the requirement to use the native `php` binary which is available in the standard DSM PATH.